.github/workflows/integration-test.yml

```
@@ -171,3 +223,4 @@ jobs:
           RUN: ${{ matrix.integration.run }}
           SKIP: ${{ matrix.integration.skip }}
           NAME: ${{ matrix.integration.name }}
+          TEST_ARGS: ${{ matrix.integration.go_test_args }}
```

actionlint (1.7.4)

property "go_test_args" is not defined in object type {name: string; os: string; run: string; secrets: array<string>; setup_scripts: array<string>; tags: string; target: string; test_args: string}

(expression)